components/SortableTable/index.vue

```diff
@@ -411,7 +418,7 @@ export default {
           <tr class="group-row">
             <td :colspan="fullColspan">
               <div class="group-tab">
-                {{ group.ref }}
+                <span class="group-tab-title-ref">{{ normalizedGroupRef }}:</span>&nbsp;{{ group.ref }}
```

Comment:
The translation should be getting used here, where it's actually displayed, instead of in the sort.  Instead of having a `normalizedGroupRef` just use the format defined by the translation (which may not be capitalized, or the group name before the "Namespace" label, etc depending on the language):

```
<div class="group-tab">
  <t :k="'resourceTable.groupBy.' + groupBy" :group="group"/>
</div>
```

```
resourceTable:
  groupBy:
    namespace: "Namespace: {group.ref}"
    project: "Project: {group.ref}"
```